ExtCore.Tests/Control.fs

```diff
@@ -38,7 +38,7 @@ type AsyncChoiceBuilderDisposeFixture() =
                 disposed.Value <- true }
 
     let createAsyncChoiceDisposable() =
```

Comment:
I think all the functions which have "Choice" word in their names should be renamed.

Reply:
I agree